### ISSUE TITLE
Segmentation fault model_from_json

### ISSUE THREAD

**Issue by derNorweg** [tensorflow/tensorflow#29588]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- Linux Ubuntu 16.04 LTS:
- TensorFlow installed with pip install:
- TensorFlow version 1.13.1
- Python version: 3.5
- CUDA/cuDNN version: 10.0/?
- GPU model and memory: Tesla K80 11GB (even after disconnecting the card from VM app still crashed)

Tensorflow loads fine, yet program crashes with segmentation error at load_from_json action.

**Code to reproduce the issue**
```
import numpy.core.multiarray
import numpy as np
import tensorflow as tf
from keras.models import load_model, model_from_json

json_file = open(os.path.join('/'.join(FLAGS.model_path.split('/')[0:-1]), 'model.json'), 'r')
loaded_model_json = json_file.read()
json_file.close()
model = model_from_json(loaded_model_json, custom_objects={'tf': tf, 'RESIZE_FACTOR': RESIZE_FACTOR})    <<<<<  This line crashes
model.load_weights(FLAGS.model_path)
```

Exactly the same code runs smoothly on my laptop with only CPU, windows 10 and python 3.7.

**Comment by jvishnuvardhan** [comment]
@derNorweg Could you provide a standalone code to reproduce the issue? You could make a gitHub gist and share the link. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
